### ISSUE TITLE
Validate the form when clicking another button?

### ISSUE THREAD

**Issue by lazharichir** [victorjonsson/jQuery-Form-Validator#486]
My page has two forms, the first one is the user registration that when clicking the submit button sends an AJAX request to register the user (wordpress.) Then, the second form is the billing form (stripe.)

So I want the first form, when clicking the submit button, to be validated. Everything works onblur but I can't check it programmatically. I am already catching the click in my .js file in order to send the AJAX request. Now, I want to validate the form before actually doing the AJAX work.

HTML:

`				<form id="be_registration_form" class="be_form be_registration_form" action="" method="POST">
					<fieldset id="be_registration_plan" class="be_registration_plan">
						<h3>Pick Your Plan</h3>
						<p>
							<button name="be_plan_monthly_btn" id="be_plan_monthly_btn" value="monthly-plan">MONTHLY!</button>
						</p>
						<p>
							<button name="be_plan_yearly_btn" id="be_plan_yearly_btn" value="yearly-plan">YEARLY!</button>
						</p>
					</fieldset>
					<fieldset id="be_registration_info" class="be_registration_info">
						<h3>Enter Your Info</h3>
						<p>
							<label for="be_user_email"><?php _e('Email'); ?></label>
							<input type="email" name="be_user_email" id="be_user_email"
								required="required">
						</p>
						<p>
							<label for="be_user_first"><?php _e('First Name'); ?></label>
							<input type="text" name="be_user_first" id="be_user_first"
								required="required">
						</p>
						<p>
							<label for="be_user_last"><?php _e('Last Name'); ?></label>
							<input type="text" name="be_user_last" id="be_user_last"
								required="required">
						</p>
						<p>
							<label for="be_user_pass"><?php _e('Password'); ?></label>
							<input type="password" name="be_user_pass" id="be_user_pass"
								required="required">
						</p>
						<p>
							<label for="be_user_pass_confirm"><?php _e('Confirm Password'); ?></label>
							<input type="password" name="be_user_pass_confirm" id="be_user_pass_confirm"
								data-validation="confirmation"
								data-validation-confirm="be_user_pass"
								required="required">
						</p>
						<p>
							<input type="hidden" name="be_plan" id="be_plan" value="">
							<input type="hidden" name="action" value="registration_user">
							<input type="hidden" name="be_register_nonce" value="<?php echo wp_create_nonce('be-register-nonce'); ?>"/>
							<button id="be_info_back" class="be_info_back"><?php _e('Back'); ?></button>
							<input id="be_info_submit" class="be_info_submit" type="submit" value="<?php _e('Next'); ?>"/>
						</p>
					</fieldset>
				</form>`

And my JS:

`$( 'input[type="submit"]' ).on( 'click', function( e ) {
	
	e.preventDefault();

	$('#be_registration_form').validate(function(valid, elem) {
	   
		if( valid ) {
			alert('valid');
			/*
			$.ajax({ 
	            data: jQuery("#be_registration_form").serialize(),
	            type: 'post',
	            datatype: 'json',
	            url: be_ajax.be_ajax_url
	        })
	        .done(function( response ) {
	        	if( response.id == 'zzz' ) {
	        		be_show_registration_panel( 'be_panel_billing' );
	        	} else {

	        	}
	    	});*/
		} else {
			alert('not valid');
		}
	});

});`

When I leave a field, it checks the field itself, but I want that:
- each field to be checked on blurred (when losing focus)
- the entire form to be validated when clicking that submit button

Any help? Thanks!

**Comment by victorjonsson** [comment]
```js
$.validate({
    modules: 'html5,security',
    form: '#be_registration_form',
    onSuccess: function() {
        // Do ajax stuff here
        return false; // preventing the form submit
    }
});
```

**Comment by lazharichir** [comment]
Awesome, works! Thanks :)
